packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -355,201 +511,332 @@ describe('meeting', () => {
         'The library has not yet been initialized',
       );
     });
-
-    it('should successfully register a handler for when live stream is changed', async () => {
-      await utils.initializeWithContext(FrameContexts.sidePanel);
-
-      let handlerCalled = false;
-      let returnedLiveStreamState: meeting.LiveStreamState | null;
-
-      meeting.registerLiveStreamChangedHandler((liveStreamState: meeting.LiveStreamState) => {
-        handlerCalled = true;
-        returnedLiveStreamState = liveStreamState;
-      });
-
-      utils.sendMessage('meeting.liveStreamChanged', { isStreaming: true });
-
-      expect(handlerCalled).toBe(true);
-      expect(returnedLiveStreamState).not.toBeNull();
-      expect(returnedLiveStreamState).toEqual({ isStreaming: true });
+    const allowedContexts = [FrameContexts.sidePanel];
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        it('should successfully register a handler for when live stream is changed', async () => {
+          await framedPlatformMock.initializeWithContext(FrameContexts.sidePanel);
```

Comment:
We need the code to pass the context change `FrameContexts.sidePanel` => `context`